enterprise/backend/test/metabase_enterprise/workspaces/test_util.clj

```diff
@@ -28,6 +30,17 @@
                                                 :model/WorkspaceDependency]
                           (tests)))))
 
+(derive :model/Workspace :model/WorkspaceCleanUpInTest)
+
+(t2/define-before-delete :model/WorkspaceCleanUpInTest
```

Comment:
Won't this hook only fire when we explicitly use this model?

Our code doesn't use this model, so I don't think it's doing anything.